Bottom: 62dcc3c6fbf83d0457fbf3b263bd8a3268b7855e
Top:    a0d7f878f73dfd0d4d6bc1521cf534a9356f9247
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:07:35 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/face3d.c b/src/hid/common/face3d.c
new file mode 100644
index 0000000..e69de29
diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
new file mode 100644
index 0000000..e69de29
